src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -1238,12 +1334,13 @@ private boolean s3JarUrlIsValid(String s3JarUrl) {
         return true;
     }
 
-    private String getBuildLogFilename() {
-        return String.format("%s-build.log", getTripPlannerString());
-    }
-
     private String getTripPlannerString() {
-        return deployment.r5 ? "r5" : "otp";
+        switch (deployment.tripPlannerVersion) {
+            case OTP_1:
+            case OTP_2:
+            default:
+                return "otp";
+        }
```

Comment:
We don't need that switch statement. This method can be turned into a constant.

Reply:
Fixed in https://github.com/ibi-group/datatools-server/pull/363/commits/d30a9edb8db1fe4acf1bc521ecae471938349f05